scripts/countries.json

```diff
@@ -0,0 +1,197 @@
+    "Indonesia": "IDN",
+    "Iran": "IRN",
+    "Iraq": "IRQ",
+    "Ireland": "IRL",
+    "Israel": "ISR",
+    "Italy": "ITA",
+    "Jamaica": "JAM",
+    "Japan": "JPN",
+    "Jordan": "JOR",
+    "Kazakhstan": "KAZ",
+    "Kenya": "KEN",
+    "Kosovo": "RKS",
+    "Kuwait": "KWT",
+    "Kyrgyzstan": "KGZ",
+    "Lannion": "FRA",
+    "Lao PDR": "LAO",
+    "Latvia": "LVA",
+    "Lebanon": "LBN",
+    "Lesotho": "LSO",
+    "Liberia": "LBR",
+    "Libya": "LBY",
+    "Lithuania": "LTU",
+    "Luxembourg": "LUX",
+    "Macedonia": "MKD",
+    "Madagascar": "MDG",
+    "Madison, WI": "USA",
+    "Malawi": "MWI",
+    "Malaysia": "MYS",
+    "Mali": "MLI",
+    "Matera (MT), Italy": "ITA",
+    "Mauritania": "MRT",
+    "Mexico": "MEX",
+    "Moldova": "MDA",
+    "Mongolia": "MNG",
+    "Montenegro": "MNE",
+    "Morocco": "MAR",
+    "Mozambique": "MOZ",
+    "Myanmar": "MMR",
+    "Namibia": "NAM",
+    "Nepal": "NPL",
+    "Netherlands": "NLD",
+    "New Caledonia": "NCL",
+    "New York, NY": "USA",
+    "New Zealand": "NZL",
+    "Nicaragua": "NIC",
+    "Niger": "NER",
+    "Nigeria": "NGA",
+    "Norrk\u00f6ping": "SWE",
+    "Norrk\u00f6ping, Sweden": "SWE",
+    "Norway": "NOR",
+    "Offenbach / Germany": "DEU",
+    "Oman": "OMN",
+    "Oslo, Norway": "NOR",
+    "Pakistan": "PAK",
+    "Palestine": "ISR",
+    "Panama": "PAN",
+    "Papua New Guinea": "PNG",
+    "Paraguay": "PRY",
+    "Peru": "PER",
+    "Philippines": "PHL",
+    "Poland": "POL",
+    "Portugal": "PRT",
+    "Puerto Rico": "PRI",
+    "Qatar": "QAT",
+    "Republic of Congo": "COG",
+    "Republic of Korea": "KOR",
+    "Reykjavik": "ISL",
+    "Romania": "ROU",
+    "Russian Federation": "RUS",
+    "Rwanda": "RWA",
+    "Saudi Arabia": "SAU",
+    "Seattle, Washington": "USA",
+    "Senegal": "SEN",
+    "Serbia": "SRB",
+    "Sierra Leone": "SLE",
+    "Slovakia": "SVK",
+    "Slovenia": "SVN",
+    "Solomon Islands": "SLB",
+    "Somalia": "SOM",
+    "Somaliland": "SOM",
+    "South Africa": "ZAF",
+    "South Sudan": "SSD",
+    "Spain": "ESP",
+    "Sri Lanka": "LKA",
+    "Sudan": "SDN",
+    "Suriname": "SUR",
+    "Swaziland": "SWZ",
+    "Sweden": "SWE",
+    "Switzerland": "CHE",
+    "Syria": "SYR",
+    "Taiwan": "TWN",
+    "Tajikistan": "TJK",
+    "Tanzania": "TZA",
+    "Tartu, Estonia": "EST",
+    "Thailand": "THA",
+    "The Gambia": "GMB",
+    "Timor-Leste": "TLS",
+    "Togo": "TGO",
+    "Toronto, Ontario, Canada": "CAN",
+    "Trinidad and Tobago": "TTO",
+    "Tromsoe, Norway": "NOR",
+    "Tunisia": "TUN",
+    "Turkey": "TUR",
+    "Turkmenistan": "TKM",
+    "Uganda": "UGA",
+    "Ukraine": "UKR",
+    "United Arab Emirates": "ARE",
+    "United Kingdom": "GBR",
+    "United States": "USA",
+    "Uruguay": "URY",
+    "Uzbekistan": "UZB",
+    "Vanuatu": "VUT",
+    "Venezuela": "VEN",
+    "Vienna": "AUT",
+    "Vietnam": "VNM",
+    "Western Sahara": "MAR",
+    "Yemen": "YEM",
+    "Zambia": "ZMB",
+    "Zimbabwe": "ZWE"
+}
```

Comment:
Can't we download and cache that from the net somewhere ?

Reply:
I mean so we skip maintaining it.

Reply:
@mraspaud This information is kept in the pycountry python package that this script depends on. This JSON cache is matching the names that contributor's use and that the shape file uses to the three letter code. So although the shapefiles should be standard, I don't depend on that. I also wouldn't expect users to have a consistent name for their countries.